Automattic/wp-calypso
# Plugins: Remove Calypsoify references

#### Changes proposed in this Pull Request

* Remove references to `Calypsoify` in Plugins components, to prevent disabling action links
* Remove references to `Calypsoify` in redirect logic for Atomic

#### Testing instructions



* Check that /plugins and individual plugin pages load for Simple sites, but you can't install or activate plugins.
* Switch to an Atomic site. You should be redirected to wp-admin for both /plugins and individual plugin pages.
* Check that /plugins and individual plugin pages load for Jetpack sites, and you can install, activate, and remove plugins.

Known issue: https://github.com/Automattic/wp-calypso/issues/54143



Related to #52986
